harvester/src/main/resources/application.properties

```diff
@@ -32,3 +32,5 @@ spring.data.mongodb.port=27017
 
 ## Login user of the mongo server.
 # spring.data.mongodb.username=
+## Logging
+logging.level.com.github.reflectoring.infiniboard.harvester=DEBUG
```

Comment:
DEBUG logging should not be activated by default. please comment this line as documentation on how to activate debug logging.